cmd/tailscale/cli/set.go

```diff
@@ -69,7 +69,7 @@ func newSetFlagSet(goos string, setArgs *setArgsT) *flag.FlagSet {
 
 	setf.StringVar(&setArgs.profileName, "nickname", "", "nickname for the current account")
 	setf.BoolVar(&setArgs.acceptRoutes, "accept-routes", false, "accept routes advertised by other Tailscale nodes")
```

Comment:
Bug: It looks like `tailscale set --accept-routes` and `tailscale up --accept-routes` don’t match:
https://github.com/tailscale/tailscale/blob/27ef9b666cd23c2ad5acb27c4f87294228219305/cmd/tailscale/cli/up.go#L107
https://github.com/tailscale/tailscale/blob/27ef9b666cd23c2ad5acb27c4f87294228219305/cmd/tailscale/cli/up.go#L79-L90

And neither of these match with ipn.NewPrefs:
https://github.com/tailscale/tailscale/blob/27ef9b666cd23c2ad5acb27c4f87294228219305/ipn/prefs.go#L662-L675

Is `tailscale up` most correct? Should we hoist `acceptRouteDefault` so it is the default for `ipn.NewPrefs`?

Reply:
@danderson and I talked about this and we agreed that `tailscale up` is Doing The Right Thing, so everything else should agree with it. I’ve addressed this change in 6f679d5bc07debf38832dcdd8af990114bff02ff.

Reply:
I’ve cherry-picked this patch to #15320, because it fixes the same underlying bug (#15319).

Reply:
Now that that PR has been merged, this one needs a rebase.

Reply:
Rebased.